src/shared/proto/tbots_software_msgs.proto

```diff
@@ -5,7 +5,7 @@ package TbotsProto;
 import "shared/proto/vision.proto";
 import "shared/proto/primitive.proto";
 import "shared/proto/tbots_timestamp_msg.proto";
-import "nanopb/options.proto";
+import "generator/nanopb/options.proto";
```

Comment:
Took a quick look through but still can't quite understand why this was needed? (I trust it was otherwise this wouldn't work, but just curious).

Reply:
`py_proto_library` doesn't work with `strip_import_prefix`